GW2EIEvtcParser/EIData/ProfHelpers/Elementalist/ElementalistHelper.cs

```diff
@@ -243,5 +244,131 @@ internal static bool IsKnownMinionID(int id)
         {
             return Minions.Contains(id);
         }
+
+        public static IReadOnlyList<AnimatedCastEvent> ComputeUpdraftCastEvents(Player player, CombatData combatData, SkillData skillData, AgentData agentData)
+        {
+            var res = new List<AnimatedCastEvent>();
+            SkillItem skill = skillData.Get(Updraft);
+            if (combatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.ElementalistUpdraft2, out IReadOnlyList<EffectEvent> updrafts))
+            {
+                foreach(EffectEvent effect in updrafts)
+                {
+                    if (effect.Duration > 0)
```

Comment:
The movement is 1 second no?
new AnimatedCastEvent(player.AgentItem, skill, effect.Time, 1000)
Should work everytime